src/main/java/uk/gov/justice/hmpps/prison/repository/MovementsRepository.java

```diff
@@ -49,12 +61,12 @@
         final var sql = MovementsRepositorySql.GET_RECENT_MOVEMENTS_BY_DATE_FOR_BATCH.getSql();
         final var types = (movementTypes == null || movementTypes.isEmpty()) ? Set.of("TRN", "REL", "ADM") : movementTypes;
 
-
-        return jdbcTemplate.query(sql,
+        final var movements = jdbcTemplate.query(sql,
```

Comment:
have to get the values and then stream them into the swagger class to avoid the performance penalty of having annotations on a row mapper class